fedimint-api/src/module/mod.rs

```diff
@@ -81,7 +81,7 @@ pub trait TypedApiEndpoint {
 
     async fn handle<'a, 'b>(
         state: &'a Self::State,
-        dbtx: fedimint_api::db::DatabaseTransaction<'b>,
+        dbtx: &'a mut fedimint_api::db::DatabaseTransaction<'b>,
```

Comment:
:heart_eyes: I disliked the owned DB tx here since forever but gave up the lifetime fight back then.